model/src/main/proto/spine_examples/chatspn/message/commands.proto

```diff
@@ -52,3 +52,19 @@ message PostMessage {
     // The message text content.
     string content = 4 [(required) = true];
 }
+
+// Tells to update a message content.
+message UpdateMessageContent {
```

Comment:
I think this command should also qualify as an "editing" command, and hence it should go to the corresponding file.